inductiva/molecules/protein_solvation/README.md

```diff
@@ -25,13 +25,13 @@ The user can further define the following simulation parameters:
                 position Langevin dynamics.
     Default value if "md".
 - `n_steps_min`: Number of steps for energy minimization. Default value is 5000.
+- `ignore_warnings`: Whether to ignore warnings during grompp (gromacs preprocessor) step. If set to False, the simulation will fail if there are warnings. If True, the simulation will run and ignore the warnings, but the results may have inaccuracies. Use with caution. 
 
-Moreover, the hardware and interaction are configured with the usual general parameters - `machine_group`, `run_async`.
- Launching a simulation returns a task object, which can be used to verify the status of the simulation, get the simulation outputs and access post-processing tools. See more in [Tasks](inductiva/README.md).
+Moreover, the hardware and interaction are configured with the usual general parameters - `machine_group`, `run_async`. Launching a simulation returns a task object, which can be used to verify the status of the simulation, get the simulation outputs and access post-processing tools. See more in [Tasks](inductiva/README.md).
 
 ### Example
```

Comment:
Following a Sarmento suggestion for the WindTunnel, I would set this example right at the top after the first introduction.